src/ser-ext-ondemandDirective.ts

```diff
@@ -541,7 +543,7 @@ class OnDemandController implements ng.IController {
                     for (const bookmark of bookmarksTyped) {
                         try {
```

Comment:
dont see a reason for that try catch here, since there is nothing which can fail. I Think the exception which will throws happens in this.destroyExistingBookmark but i think we should handle the error not here, it is in general not a good practice to have an empty catch block where we just do nothing, so nobody will knows something fails.

Reply:
you are correced, it is not unnecessary and is removed in new commit

Reply:
sounds good to me, and looks good 👍